cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -171,7 +172,11 @@ def set_agent_creds(appliance, request, provider):
     ssa_agent_view = view.browser.create_view(EC2EndpointForm)
     ssa_agent_view.smart_state_docker.fill({'username': username, 'password': password})
     ssa_agent_view.default.validate.click()
+    # TODO: Remove time.sleep asap BZ fix.
+    if BZ(1608023, forced_streams=['5.9', '5.10']).blocks:
+        time.sleep(35)
```

Comment:
gross.

Reply:
:(

Reply:
I've had to do it in the auth tests too, at least its wrapped in a BZ.